web3/providers/async_ipc.py

```diff
@@ -133,12 +136,12 @@ async def make_request(self, method: RPCEndpoint, params: Any) -> RPCResponse:
                 sock.sendall(request)
 
             raw_response = b""
-            async with Timeout(self.timeout) as timeout:
+            async with asyncio.timeout(self.timeout) as timeout:
```

Comment:
Why did you replace Timeout from `web3._utils.threads` on Timeout from `asyncio`?